db/seeds.rb

```diff
@@ -35,12 +35,38 @@
   a.price_per_day = rand(25..75)
   a.height = rand(81..99)
   a.weight = rand(48..84)
-  a.color = AlpacasController::COLORS.sample
-  a.wool_type = AlpacasController::WOOL_TYPES.sample
+  a.color = Alpaca::COLORS.sample
+  a.wool_type = Alpaca::WOOL_TYPES.sample
   a.owner = User.order(Arel.sql('RANDOM()')).first
   a.save
 end
 
+a = Alpaca.new
+a.name = "Oldman"
+a.nick_name = "Opa"
+a.age = 40
+a.price_per_day = 56
+a.height = 89
+a.weight = 78
+a.color = "gray"
+a.wool_type = "suri"
+a.address = "balanstrasse 74 Munich"
+a.owner = User.order(Arel.sql('RANDOM()')).first
+a.save
+
+a = Alpaca.new
+a.name = "Joe"
+a.nick_name = "Jojo"
+a.age = 40
+a.price_per_day = 56
+a.height = 89
+a.weight = 78
+a.color = "gray"
+a.wool_type = "suri"
+a.address = "3103 Broadway, New York, NY 10027, USA"
+a.owner = User.order(Arel.sql('RANDOM()')).first
+a.save
```

Comment:
seed the address for all the alpacas. eg.: Faker::Address.city and not only for 2